apps/theming-designer/src/components/FabricPalette.tsx

```diff
@@ -73,66 +49,220 @@ export const FabricPalette: React.StatelessComponent<IFabricPaletteProps> = (pro
         </thead>
         <tbody>
           <tr>
-            <Text as="td">{fabricSlotWidget(FabricSlots.themeDarker)}</Text>
-            <Text as="td">{props.themeRules![FabricSlots[FabricSlots.themeDarker]].color!.str}</Text>
-            <Text as="td">{fabricSlotWidget(FabricSlots.black)}</Text>
-            <Text as="td">{props.themeRules![FabricSlots[FabricSlots.black]].color!.str}</Text>
-            <Text as="td">{fabricSlotWidget(FabricSlots.neutralTertiaryAlt)}</Text>
-            <Text as="td">{props.themeRules![FabricSlots[FabricSlots.neutralTertiaryAlt]].color!.str}</Text>
+            <Text as="td">
+              <FabricSlotWidget
+                themeRules={props.themeRules}
+                slot={FabricSlots.themeDarker}
+                onFabricPaletteColorChange={props.onFabricPaletteColorChange}
```

Comment:
>onFabricPaletteColorChange={props.onFabricPaletteColorChange} [](start = 16, length = 61)

nit but i would be so much more concise (and result in smaller bundle sizes from allowing "uglification" of the variables) if you destructure all of these props.*, in turn allowing you to use do `<FabricSlotWidget onFabricPaletteColorChange ... />` here and in all the lines below (and in other files too)